simulation_parameters/microbe_stage/organelles.json

```diff
@@ -137,39 +137,6 @@
     "EditorButtonOrder": 5,
     "EndosymbiosisUnlocks": "nitrogenfixingplastid"
   },
-  "protoplasm": {
```

Comment:
Removing an organelle type necessitates a save breakage point so I'd prefer to avoid it if not absolutely required.